app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/NavigationOverlayFragment.kt

```diff
@@ -82,6 +90,7 @@ enum class NavigationEvent {
     }
 }
 
+@Suppress("LargeClass")
```

Comment:
Wonder if it's time to break up the large classes. File?

Reply:
Definitely.  I think it's out of scope for this (already huge) PR though.